OrbitCore/OrbitModule.h

```diff
@@ -15,7 +15,8 @@ class Pdb;
 
 //-----------------------------------------------------------------------------
 struct Module {
-  Module();
+  Module(){};
+  Module(const std::string, uint64_t, uint64_t);
```

Comment:
This is an Address range and I gave it proper names. I used a local AddressRange struct in LinuxUtils now, mainly to avoid std::pair. 
A struct here instead of having the member variables `m_AdressStart` and `m_AdressEnd` would definitely be cleaner, but I assume it requires changes in a lot of places. I am gonna leave it as two values for now